src/fusion_power_plant.cc

```diff
@@ -196,9 +201,13 @@ bool FusionPowerPlant::ReadyToOperate() {
   // determine required tritium storage inventory
   double required_storage_inventory = fuel_usage_mass + SequesteredTritiumGap();
 
-  // check tritium storage quantity
+  // check basic tritium storage quantity requirement
   if (tritium_storage.quantity() < required_storage_inventory || !TritiumStorageClean()) {
-        return false;
+    return false;
+  }
+  // insure correct startup behavior
+  if (tritium_storage.quantity() < startup_inventory && sequestered_tritium->quantity() < cyclus::eps_rsrc()) {
```

Comment:
Maybe this should be first, and perhaps change the order of the conditions.  The order doesn't matter for logic reasons, but communicates the purpose - we only care about this condition when the sequestered tritium is essentially 0

Reply:
Sure, that makes sense to me.